tidb-cloud/migrate-from-mysql-using-data-migration.md

```diff
@@ -100,10 +100,37 @@ Before creating a migration job, set up the network connection according to your
 
 - If you use public IP (this is, standard connection) for network connection, make sure that the upstream database can be connected through the public network.
 
-- If you use VPC Peering, set it up according to [Add VPC peering requests](/tidb-cloud/set-up-vpc-peering-connections.md#step-1-add-vpc-peering-requests).
-
 - If you use AWS PrivateLink, set it up according to [Set Up Private Endpoint Connections](/tidb-cloud/set-up-private-endpoint-connections.md).
 
+- If you use VPC Peering, see the following instructions to configure the network.
+
+<details>
+<summary> Set up VPC Peering</summary>
+
+Make sure that your TiDB Cluster can connect to the MySQL service.
+
+If your MySQL service is in an AWS VPC that has no public internet access, take the following steps:
+
+1. [Set up a VPC peering connection](/tidb-cloud/set-up-vpc-peering-connections.md) between the VPC of the MySQL service and your TiDB cluster.
+2. Modify the inbound rules of the security group that the MySQL service is associated with. 
+
+    You must add [the CIDR of the region where your TiDB Cloud cluster is located](/tidb-cloud/set-up-vpc-peering-connections.md#prerequisite-set-a-project-cidr) to the inbound rules. Doing so allows the traffic to flow from your TiDB Cluster to the MySQL instance.
+
+3. If the MySQL URL contains a hostname, you need to allow TiDB Cloud to be able to resolve the DNS hostname of the MySQL service. 
+
+    1. Follow the steps in [Enable DNS resolution for a VPC peering connection](https://docs.aws.amazon.com/vpc/latest/peering/modify-peering-connections.html#vpc-peering-dns).
+    2. Enable the **Accepter DNS resolution** option.
+
+If your MySQL service is in a GCP VPC that has no public internet access, take the following steps:
```

Comment:
DM does not support  GCP yet, We plan to  support before  March 31,2023. Please  hide the following content about GCP now.

Reply:
OK commented out for now.